### PR TITLE
Add ipv6 support in is_local_interface function

### PR DESCRIPTION
**Pull request by miguelxpn**

### CODE REVIEW

File: webapp/graphite/util.py [hunk 1]
```diff
@@ -72,13 +72,20 @@ def getProfileByUsername(username):
...
     return None
 
-
 def is_local_interface(host):
+  is_ipv6 = False
   if ':' in host:
-    host = host.split(':',1)[0]
+    try:
+      socket.inet_pton(socket.AF_INET6, host)
+      is_ipv6 = True
```
Comment (obfuscurity) [inline webapp/graphite/util.py]: Don't we need to manipulate `host` here for ipv6?

Comment (miguelxpn) [inline webapp/graphite/util.py]: Not really, if host is already a valid ipv6 address then we don't need to manipulate it further, if it's not then socket.error will be raised and we'll split the port out of it and then everything will proceed as it did before, should work unless there's something I'm missing.

**Comment by miguelxpn** [comment]
Turns out I had actually missed something, I changed the code and added support for bracket notation so that the function works with ipv6 addresses with a port. Hopefully that's it now.

File: webapp/graphite/util.py (hunk 1 above)
Comment (obfuscurity) [inline webapp/graphite/util.py]: Ok, I'm trusting you here since I have little practical ipv6 experience to compare against. :smile_cat:

**Comment by obfuscurity** [comment]
Thanks!

**Pull request merged**